cranelift/README.md

```diff
@@ -85,6 +84,13 @@ Building Cranelift
 Cranelift uses a [conventional Cargo build
 process](https://doc.rust-lang.org/cargo/guide/working-on-an-existing-project.html).
 
+As a user of wasmtime and or Cranelift you can use stable rust 1.43 or newer.
+However if you want to develop Cranelift, you'll need a nightly version of
+Rust, as some tests use unstable features.
```

Comment:
Not for Cranelift, only Wasmtime. Lightbeam is a part of Wasmtime. I haven't ever used nightly rustc to work on Cranelift.